openapi/preprocess_spec.py

```diff
@@ -96,10 +96,12 @@ def remove_watch_operations(op, parent, operation_ids):
 
 def strip_tags_from_operation_id(operation, _):
     operation_id = operation['operationId']
-    for t in operation['tags']:
-        operation_id = operation_id.replace(_to_camel_case(t), '')
-    operation['operationId'] = operation_id
-
+    try:
+        for t in operation['tags']:
+            operation_id = operation_id.replace(_to_camel_case(t), '')
+        operation['operationId'] = operation_id
+    except KeyError:
```

Comment:
Rather than this, shouldn't we just:

```python
if 'tags' in operation:
   for t in operation['tags']:
...
```

I don't really like the "catch exception as an if statement" approach.